lib/settings.coffee

```diff
@@ -1,8 +1,15 @@
 module.exports =
   config:
+
+    fontSize:
+      title: 'Font Size'
+      description: 'Change the UI font size. Needs to be between 10 and 20.'
+      type: 'string'
```

Comment:
I'm pretty certain that if you don't specify a default, the default will be `undefined`.